### ISSUE TITLE
Light version

### ISSUE THREAD

**Issue by ignacy** [aktersnurra/no-clown-fiesta.nvim#40]
Thanks for making this! 

I'd love to have a light version for when I work in the sun.. I tried to make some edits myself but because the code is quite modularized it's not easy - even when you "reverse" the pallete.lua there are mappings of color names to the way they are actually used in different files. Would you consider adding a light version? Or maybe you could give me some tips on making the edits easier?

**Comment by aktersnurra** [comment]
Agreed, I have been thinking about this for a while, I will try to have a look at this during this weekend.

**Comment by tkgalk** [comment]
I know you don't have any obligation to the anonymous users online, but any progress on this one? Would you need some help? I'd love a light-theme too. :)

**Comment by aktersnurra** [comment]
> I know you don't have any obligation to the anonymous users online, but any progress on this one? Would you need some help? I'd love a light-theme too. :)

Sorry, no progress, life go in the way. I will try to make a push before the summer since I would like this feature myself as well, now that it is so light outside.

**Comment by aktersnurra** [comment]
I finally made some [progress](https://github.com/aktersnurra/no-clown-fiesta.nvim/commit/a70eef2af24f804cc017bb1b03ee238b033bffae). I will improve it in the coming days.
